tests/test_compressors/model_compressors/test_model_compressor.py

```diff
@@ -187,37 +170,14 @@ def create_quantization_config(bits=8, type="int", strategy="tensor"):
         create_quantization_config(bits=8, type="float", strategy="channel"),
     ],
 )
-def test_composability(
-    tmp_path, fake_model_class, sparsity_config, quantization_config
-):
-    from compressed_tensors.quantization.lifecycle.forward import quantize
-
-    weights = torch.rand(10, 5)
-    sparse_weights = induce_sparsity(weights, sparsity_config.global_sparsity)
-
-    quantization_args = quantization_config.config_groups["group_0"].weights
-
-    if quantization_args.strategy == "channel":
-        scale = torch.ones((weights.shape[0], 1))
-    elif quantization_args.strategy == "tensor":
-        scale = torch.tensor([1.0])
-
-    zero_point = torch.zeros_like(scale)
-
-    quantized_weights = quantize(
-        sparse_weights,
-        scale=scale,
-        zero_point=zero_point,
-        args=quantization_args,
-    )
+def test_composability(tmp_path, sparsity_config, quantization_config):
```

Comment:
Essentially refactored the model creation logic into a function that was re-used in a newly added test